tasks/push_cacher_image.yml

```diff
@@ -4,7 +4,7 @@
     state: stopped
 
 - name: Commit container
-  command: docker commit "{{ INFINISPAN_SERVER_NAME }}" "{{ CACHER_RELEASE_IMAGE_NAME }}"
+  command: docker commit "{{ INFINISPAN_SERVER_NAME }}" "{{ CACHER_RELEASE_REG_INAME }}:{{ DOCKER_BUILD_TAG }}"
 
 - name: Push Container To Docker Hub
-  command: docker push "{{ CACHER_RELEASE_IMAGE_NAME }}"
+  command: docker push "{{ CACHER_RELEASE_REG_INAME }}:{{ DOCKER_BUILD_TAG }}"
```

Comment:
This shouldn't need the `:{{ DOCKER_BUILD_TAG }}`
it is being set in the env
`CACHER_RELEASE_IMAGE_NAME: "{{ CACHER_RELEASE_REG_INAME }}:{{ DOCKER_PULL_TAG }}"`

Reply:
or switch it back to `CACHER_RELEASE_IMAGE_NAME`

Reply:
As per our discussion on slack: `DOCKER_BUILD_TAG`  != `DOCKER_PULL_TAG`. I did move all image variable definitions into [shared_variables.yml](./environments/shared_variables.yml) now.